source/onboard/mattermost-limits.rst

```diff
@@ -14,19 +14,21 @@ In addition to these limits, visit our pricing page to see a complete list of fe
 
    These limits do not apply to self-hosted deployments. 
 
-Mattermost Free limits
-----------------------
+Mattermost Cloud Free limits
+----------------------------
 
 **Platform limits**
 
-- Maximum 1 team. Any additional teams created during a trial or on a paid plan will be archived.
```

Comment:
@johndavidlugtu - Can you please confirm that this line should be removed from the docs please?

Reply:
@cwarnermm 10GB file storage limit will now be 1GB as per this [post](https://community.mattermost.com/private-core/pl/o4nohhu1zjbtmcpqo9ekx8x8nr)

Will get back to you re: team limits.

Reply:
Hey @cwarnermm !

Team limit is kept at 1 team

cc: @laneycs

Reply:
Thanks, @johndavidlugtu! Brought these bullets back and updated them where needed.